index.d.ts

```diff
@@ -0,0 +1,11 @@
+declare namespace CBOR {
+  export function encode(data: any, format?: "buffer"): Buffer;
```

Comment:
Looks like the `buffer` form can also return a string:

https://github.com/ARMmbed/cbor-sync/blob/master/main.js#L518

Reply:
No, it cannot. When `format` is "buffer" or `undefined`,

https://github.com/ARMmbed/cbor-sync/blob/0b3dcad537433f4099562dd549681abdb988aa39/main.js#L533-L537
Here `BufferWriter` is constructed with `stringFormat=undefined`.

https://github.com/ARMmbed/cbor-sync/blob/0b3dcad537433f4099562dd549681abdb988aa39/main.js#L518
`result.toString` would not be executed.